src/port_channel.cc

```diff
@@ -67,7 +67,7 @@ MSCCLPP_API_CPP void ProxyService::bindThread() {
 
 ProxyHandlerResult ProxyService::handleTrigger(ProxyTrigger triggerRaw) {
   ChannelTrigger* trigger = reinterpret_cast<ChannelTrigger*>(&triggerRaw);
-  std::shared_ptr<Host2DeviceSemaphore> semaphore = semaphores_[trigger->fields.chanId];
+  std::shared_ptr<Host2DeviceSemaphore> semaphore = semaphores_[trigger->fields.semaphoreId];
```

Comment:
A bit confused about this field. Here if we want to put data, we need get semaphore object, then use the connection from semaphore to send data.
Maybe use channel is easier to understand. Get use the connection from channel to send data.